srcpkgs/docker-cli/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'docker-cli'
+pkgname=docker-cli
+version=20.10.5
+revision=1
+archs="aarch64* armv[67]l* x86_64* ppc64le*"
+create_wrksrc=yes
+build_style=go
+go_import_path="github.com/docker/cli"
+hostmakedepends="pkg-config tar"
+depends="containerd runc iptables xz git"
+short_desc="Pack, ship and run any application as a lightweight container - cli"
+maintainer="PWA COLLECTIVE <mail@pwa.cya.cx>"
+license="Apache-2.0"
+homepage="http://www.docker.io"
+distfiles="https://github.com/docker/cli/archive/v${version}.tar.gz"
+checksum=4ba845f8c7e2e0a2ca1ec6589847159ca8d0d37b609f0e6f78def7a893b9b342
+system_groups="docker"
+
+_cli_commit=55c4c88966a912ddb365e2d73a4969e700fc458f
+_cli_builddir="cli-$version"
```

Comment:
Not necessary with the `create_wrksrc` drop.

Reply:
Implemented.